### PR TITLE
2.1 beta blog post

### PR DESCRIPTION
**Pull request by mixonic**
Draft of the 2.1 beta blog post. [Rendered](https://github.com/mixonic/website/blob/blog-2.1/source/blog/2015-08-16-ember-2-1-beta-released.md)

### CODE REVIEW

**Comment by rwjblue** [comment]
It might be nice to credit @jmurphyau for creating ember-get-helper and helping us with the integration efforts.

File: source/blog/2015-08-16-ember-2-1-beta-released.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+* `next` is a caller for the previously registered handler. Often, this is the
+  default handler.
+* `options.id` is an id in the format `package-name.specific-deprecation`
+
+Deprecation handlers will also be provided:
+
+* `options.until` is the version of Ember this feature and deprecation will be
+  removed in
+
+As of Ember 1.13, `deprecate` and `warn` calls must provide an `id` option,
```
Comment (rwjblue) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: Ember 1.13 does not include `id` and `until`.  The rule began in 2.0.0.

File: source/blog/2015-08-16-ember-2-1-beta-released.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  `registerOptionsForType`, `registeredOptionsForType` which manage options
+  for a factory (is it a singleton, can it be instantiated).
+
+`Ember.ApplicationInstance` instances are passed as the first argument to
+`instanceInitializer` hooks in 2.1. `instanceInitializer` hooks are where
+objects and classes can be fetched out of the configured and booted application.
+Two new relevent public APIs will exist on `Ember.ApplicationInstance`s:
+
+* `lookup` - fetch an instance of a factory (with dependencies)
+* `lookupFactory` - fetch a factory (with dependencies)
```
Comment (rwjblue) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: This is not public (and is underscored as `_lookupFactory`).

**Comment by rwjblue** [comment]
I just pushed the 2.1.0-beta.1 CHANGELOG entries. Only two were added:

- [#11440](https://github.com/emberjs/ember.js/pull/11440) [DEPRECATION] Deprecate using `instance.container.lookup` on first argument to `instanceInitializers`. Use `instance.lookup` instead.
- [#11440](https://github.com/emberjs/ember.js/pull/11440) [DEPRECATION] Deprecate passing two arguments to an initializers `initialize` function.

File: source/blog/2015-08-16-ember-2-1-beta-released.md (hunk 2 above)
Comment (mixonic) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: Was that changed? I'm looking at https://github.com/emberjs/ember.js/pull/11440/files#diff-b94da47ec2a2dead4c182caa3fae3ae3R68

Comment (mixonic) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: Ah private in master, gotcha: https://github.com/emberjs/ember.js/blob/master/packages/ember-runtime/lib/mixins/container_proxy.js#L67

**Comment by mixonic** [comment]
Now with a mention of the deprecations and with credit.

File: source/blog/2015-08-16-ember-2-1-beta-released.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+Ember.js 2.1 will be a minor release of Ember. This means changes to the API are
+made in an additive, backwards compatible manner. In roughly six weeks, these
+features will be part of the 2.1 stable version.
+
+A summary of the new features in today's release follows.
+
+#### `{{get}}` Helper
+
+The `{{get}}` helper allows dynamic property lookup on objects in templates.
+For example, these two usages are equivilent:
```
Comment (lukemelia) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: s/equivilent/equivalent/

Comment (mixonic) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: Fixed, thanks Luke :-)

File: source/blog/2015-08-16-ember-2-1-beta-released.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+[RFC #46](https://github.com/emberjs/rfcs/blob/master/text/0046-registry-reform.md)
+and the initial implementation in
+[PR #11440](https://github.com/emberjs/ember.js/pull/11440). To better
+understand dependency management in Ember and how to use these APIs, see the
+section on [dependency management](http://guides.emberjs.com/v1.13.0/understanding-ember/dependency-injection-and-service-lookup/#toc_dependency-management-in-ember-js) in
+the 1.13 guides.
+
+This feature also introduces two minor deprecations:
+
+* Calling `appInstance.controller.lookup` on the first argument to an instance
```
Comment (lukemelia) [inline source/blog/2015-08-16-ember-2-1-beta-released.md]: I think you intended `appInstance.container.lookup`?

**Pull request merged**